drivers/src/pci/config_space.rs

```diff
@@ -38,19 +38,18 @@ impl PciConfigSpace {
         }
     }
 
-    /// Returns an iterator across the top-level buses.
-    pub fn busses(&self) -> HostControllersIter {
+    /// Returns the top-level bus.
```

Comment:
Not related, but in the commit message, "buses" might be a more common plural form.

Reply:
Done.